Sources/Components/Broadcast/BroadcastContainerView/BroadcastContainerView.swift

```diff
@@ -0,0 +1,258 @@
+
+        addSubview(contentView)
+
+        let insets = LayoutConstants.ContentView.insets
+
+        NSLayoutConstraint.activate([
+            contentView.topAnchor.constraint(equalTo: topAnchor, constant: insets.top),
+            contentView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: insets.leading),
+            contentView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: insets.trailing),
+        ])
+
+        contentView.setContentCompressionResistancePriority(.defaultLow, for: .vertical)
+        contentView.setContentHuggingPriority(.defaultHigh, for: .vertical)
+    }
+
+    func layoutBroadcastViews(from broadcasts: [Broadcast]) {
+        let broadcastViews = broadcasts.map { broadcastView(from: $0) }
+
+        broadcastViews.forEach { add($0, to: contentView) }
+    }
+
+    func broadcastView(from broadcast: Broadcast) -> BroadcastView {
+        let broadcastView = BroadcastView(frame: .zero)
+
+        // Set the message on the broadcast view and prepare it in its expanded state
+        broadcastView.present(message: broadcast.message, animated: false)
+
+        return broadcastView
+    }
+
+    func add(_ broadcastView: BroadcastView, to stackView: UIStackView) {
+        broadcastView.translatesAutoresizingMaskIntoConstraints = false
+        broadcastView.isHidden = true
+
+        stackView.addArrangedSubview(broadcastView)
+
+        NSLayoutConstraint.activate([
+            broadcastView.leadingAnchor.constraint(equalTo: stackView.leadingAnchor),
+            broadcastView.widthAnchor.constraint(equalTo: stackView.widthAnchor),
+        ])
+
+        broadcastView.setContentCompressionResistancePriority(.defaultLow, for: .vertical)
+        broadcastView.setContentHuggingPriority(.defaultHigh, for: .vertical)
+
+        let tapRecogninzer = UITapGestureRecognizer(target: self, action: #selector(broadcastViewTapped(_:)))
+        broadcastView.addGestureRecognizer(tapRecogninzer)
+
+        broadcastView.layoutIfNeeded()
+    }
+
+    func remove(_ broadcastView: BroadcastView) {
+        guard let subView = contentView.arrangedSubviews.filter({ $0 == broadcastView }).first else {
+            return
+        }
+
+        contentView.removeArrangedSubview(subView)
+        subView.removeFromSuperview()
+    }
+
+    func reset() {
+        contentView.arrangedSubviews.forEach { view in
+            contentView.removeArrangedSubview(view)
+            view.removeFromSuperview()
+        }
+    }
+
+    @objc private func broadcastViewTapped(_ sender: UIGestureRecognizer) {
+        guard let broadcastView = sender.view as? BroadcastView else {
+            return
+        }
+
+        if let delegate = delegate {
+            let newContainerSize: CGSize = {
+                let width = frame.width
+                if contentView.arrangedSubviews.count == 1 { // This is the last broadcast that soon will be removed
```

Comment:
We could use a named variable instead of a comment here.

For example:
```swift
let isBroadcastToBeRemoved = contentView.arrangedSubviews.count == 1
if isBroadcastToBeRemoved {
```